### ISSUE TITLE
Integrate track progress bar

### ISSUE THREAD

**Issue by fenryxo** [tiliado/nuvolaplayer#155]
Feedback from @tcitworld:

> > 4) Which feature are you missing in Nuvola Player most?

> Full control through Gnome Media Player indicator (can be buggy).
> [...]
> I remember the progress bar wasn't enabled, at least for the Deezer plugin.

**Comment by tcitworld** [comment]
I got the chance to test it in cinnamon and the same problem happens.

**Comment by fenryxo** [comment]
> Moreover, album title is always marked as « unknown ».

That's correct because Deezer doesn't show album name in its web interface.

**Comment by tcitworld** [comment]
Oh. OK. Didn't noticed this with other desktop environments.

**Comment by fenryxo** [comment]
Related MPRIS API:
* [Metadata](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Property:Metadata) - `mpris:length`
* [Position](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Property:Position)
* [CanSeek](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Property:CanSeek)
* [Seek()](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Method:Seek)
* [SetPosition()](https://specifications.freedesktop.org/mpris-spec/latest/Player_Interface.html#Method:SetPosition)

**Comment by fenryxo** [comment]
All required APIs have been added to the master branch (Nuvola 4.4.13). Remaining tasks:

- [ ] Write documentation for maintainers.
- [ ] Integrate progress bar in my streaming services
- [ ] Ask maintainers of the other services to integrate progress bar.
